proto/common.proto

```diff
@@ -59,6 +59,9 @@ message WorkerNode {
   // TODO #8940 `parallel_units` should be moved into `Property`
   repeated ParallelUnit parallel_units = 5;
   Property property = 6;
+
+  // Ranges from 0 to 1023, used to generate the machine ID field in the global unique ID.
+  optional uint32 transactional_id = 7;
```

Comment:
Since a number of users (including our cloud) have officially started using Risingwave, many of worker IDs have already exceeded 1024. To ensure compatibility, we have no choice but to use a single field. 😢